paasta_tools/metrics/metastatus_lib.py

```diff
@@ -32,9 +32,10 @@
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import print_with_indent
 
-
+class ResourceInfo(namedtuple('ResourceInfo', ['cpus', 'mem', 'disk', 'gpus'])):
```

Comment:
Instead of inheriting from namedtuple, can we just make cpus, mem, disk and gpus class members?